spec/spec_helper.rb

```diff
@@ -96,6 +97,8 @@
   Kernel.srand config.seed
 =end
 
-  config.include RequestSpecHelper, type: :request
   config.include FactoryBot::Syntax::Methods
+  config.include UserHeaderSpecHelper
```

Comment:
Why do you need this `UserHeaderSpecHelper`? Like @syncrou said below, one of them should be enough, right?

Reply:
We do the same thing on Catalog - I was just trying to get it to work, but it still isn't. I'll remove them both as they don't do anything it appears.

Reply:
Same thing here, no need to include `UserHeaderSpecHelper`